server/genserver.go

```diff
@@ -95,22 +98,20 @@ func (s *Server) Serve() error {
 		s.l.Info("please provide a kvconfig.yml file to enable app log routing")
 	}
 
-	if tracingToken := os.Getenv("TRACING_ACCESS_TOKEN"); tracingToken != "" {
-		ingestUrl := os.Getenv("TRACING_INGEST_URL")
-		if ingestUrl == "" {
-			ingestUrl = "https://ingest.signalfx.com/v1/trace"
+	tracingToken := os.Getenv("TRACING_ACCESS_TOKEN")
+	isLocal := os.Getenv("_IS_LOCAL") == "true"
+	if tracingToken != "" || isLocal {
+		ingestURL := os.Getenv("TRACING_INGEST_URL")
+		if ingestURL == "" {
+			ingestURL = "https://ingest.signalfx.com/v1/trace"
 		}
 
-		// Create a Jaeger HTTP Thrift transport
-		transport := transport.NewHTTPTransport(ingestUrl,
-			transport.HTTPBasicAuth("auth", tracingToken))
-
 		// Add rate limited sampling. We will only sample [Param] requests per second
 		// and [MaxOperations] different endpoints. Any endpoint above the [MaxOperations]
 		// limit will be probabilistically sampled.
 		cfgSampler := &jaegercfg.SamplerConfig{
-			Type: jaeger.SamplerTypeRateLimiting,
-			Param: 5,
+			Type:          jaeger.SamplerTypeRateLimiting,
+			Param:         5,
```

Comment:
Note: scope creep. I vaguely recall @mohit saying he wanted to open the flood gates for tracing now that we have a tracing plan w/ SignalFX. At the very least, it might make sense to increase the sampling rate.

Reply:
Agree. I think we should sample using [GuaranteedThroughputProbabilisticSampler](https://godoc.org/github.com/uber/jaeger-client-go#GuaranteedThroughputProbabilisticSampler) and capture significantly more than 5/sec